src/layout/ImageUpload/ImageDropzone.module.css

```
@@ -0,0 +1,95 @@
+.placeholder {
+  background-image:
+    linear-gradient(
+      45deg,
+      rgba(204, 204, 204, 0.5) 25%,
+      transparent 25%,
+      transparent 75%,
+      rgba(204, 204, 204, 0.5) 75%,
+      rgba(204, 204, 204, 0.5) 100%
+    ),
+    linear-gradient(
+      45deg,
+      rgba(204, 204, 204, 0.5) 25%,
+      rgba(255, 255, 255, 0.5) 25%,
+      rgba(255, 255, 255, 0.5) 75%,
+      rgba(204, 204, 204, 0.5) 75%,
+      rgba(204, 204, 204, 0.5) 100%
+    );
+  background-size: 20px 20px; /* Adjust size of each square */
+  background-position:
+    0 0,
+    10px 10px; /* Offset the second gradient */
+  height: 320px;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: var(--ds-size-4);
+  border-radius: var(--ds-border-radius-lg);
+  border: var(--ds-border-width-default) solid var(--ds-color-border-subtle);
+  box-sizing: border-box;
+}
+
+.placeholder:hover {
+  background-image:
+    linear-gradient(
+      45deg,
+      rgba(204, 204, 204, 0.75) 25%,
+      transparent 25%,
+      transparent 75%,
+      rgba(204, 204, 204, 0.75) 75%,
+      rgba(204, 204, 204, 0.75) 100%
+    ),
+    linear-gradient(
+      45deg,
+      rgba(204, 204, 204, 0.75) 25%,
+      rgba(255, 255, 255, 0.75) 25%,
+      rgba(255, 255, 255, 0.75) 75%,
+      rgba(204, 204, 204, 0.75) 75%,
+      rgba(204, 204, 204, 0.75) 100%
+    );
+
+  .dropZone {
+    border-style: solid;
+  }
+}
```

```suggestion
.placeholder:hover, .dragActive {
  background-image:
    linear-gradient(
      45deg,
      rgba(204, 204, 204, 0.75) 25%,
      transparent 25%,
      transparent 75%,
      rgba(204, 204, 204, 0.75) 75%,
      rgba(204, 204, 204, 0.75) 100%
    ),
    linear-gradient(
      45deg,
      rgba(204, 204, 204, 0.75) 25%,
      rgba(255, 255, 255, 0.75) 25%,
      rgba(255, 255, 255, 0.75) 75%,
      rgba(204, 204, 204, 0.75) 75%,
      rgba(204, 204, 204, 0.75) 100%
    );

  .dropZone {
    border-style: solid;
  }
}
```

Instead of repeating the styling for dragActive class, it can be called in the same as placeholder:hover 🙂

---

Ofcourse! Nice catch 🥇